src/components/FlexibleForm/ControlDecorators/withReferenceValues/withReferenceValues.css

```diff
@@ -6,11 +6,12 @@
   & .options-dropdown {
     display: flex;
     height: 100%;
-    margin: 25.5px 0 0 -1px;
+    margin: 25px 0 0 -1px;
```

Comment:
I see that these changes broke styles in some fields (see screenshot). I'm not able to explain why it happens, but `25.5px` solves this problem.
![image](https://user-images.githubusercontent.com/55138456/83155566-0abb1880-a10a-11ea-91bf-d90b26777c42.png)

Reply:
It;s look like I managed issue with vertical decorator alignment in another way